### DISCUSSION TITLE
Using @auth/sveltekit, Okta error invalid_request

### DISCUSSION THREAD

**Discussion by SQLSourcerer** [nextauthjs/next-auth#6885] [Help]
As of this writing, I'm using @auth/core 0.5.1 and @auth/sveltekit 0.3.0

The authorization call was redirecting me to http://localhost:5173/auth/callback/okta?error=invalid_request&error_description=The+authentication+request+has+an+invalid+%27state%27+parameter

I was using the following configuration for Okta in hooks.server.ts:
```
Okta({ 
  clientId: env.OKTA_CLIENT,
  clientSecret: env.OKTA_SECRET,
  issuer: `https://${env.OKTA_DOMAIN}/oauth2/default`
})
```

It turns out "state" needs to be added to the array of checks.

```
Okta({ 
  clientId: env.OKTA_CLIENT,
  clientSecret: env.OKTA_SECRET,
  issuer: `https://${env.OKTA_DOMAIN}/oauth2/default`,
  checks: ["pkce", "state"]
})
```

It seems simple in retrospect, but it took me several days to figure it out

**Comment by ianschmitz** [comment]
Yes i ran into this as well @SQLSourcerer. I think this should be filed as a bug.
